src/test/java/org/sonatype/nexus/ci/config/Nxrm3ConfigurationTest.groovy

```diff
@@ -37,6 +37,9 @@ class Nxrm3ConfigurationTest
   def 'it checks nxrm version'() {
     given:
       URL.metaClass.getText = {
+        if (delegate.path.startsWith('//')) {
```

Comment:
Question: Can you help me understand what this added statement is here for?

Reply:
Sure! This is checking for a call to the status service with a URL that has two consecutive slashes in it, which is the condition causing this call to fail if you have a trailing slash on your server URL.

Reply:
Would it not be that the url 'contains' consecutive slashes rather than ending with consecutive slashes?